src/space/objects/realization/blackhole.js

```diff
@@ -16,7 +16,7 @@ export default class BlackHole(){
     this.x = x;
     this.y = y;
     this.bhImage = new Image();
-    this.bhImage = './resources/space/black-hole.png';
+    this.bhImage.src = './resources/space/black-hole.png';
```

Comment:
As I mentioned in my previous comment - could you please consider using our utility class called _AssetLoader_? I am afraid that if all the asset loading is not in one place, we might forget this change while trying to merge our group branch with others. Thank you